.github/workflows/lint-test-cross.yml

```
@@ -24,7 +24,43 @@
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: setup node
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: 'package.json'
+
+      - run: npm i -g @antfu/ni
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)